test/test_util.h

```diff
@@ -542,11 +542,10 @@ enum GenMode
  * Initialize value
  */
 template <typename T>
-__host__ __device__ __forceinline__ void InitValue(GenMode gen_mode, T &value, int index = 0)
```

Comment:
Isn't it used in the host/device method of `struct TransformOp` in test/test_iterator.cu?

Reply:
Regarding test/test_iterator.cu. It seems to be using deprecated methods for TestTex*.

Reply:
> https://github.com/NVIDIA/cub/pull/338

Yeah -_- CLion missed that usage. I'll have to approach this differently. I'll probably just #ifdef out the random usage that the compiler was complaining about and add some warnings.

> Regarding test/test_iterator.cu. It seems to be using deprecated methods for TestTex*.

This is fine. We build tests with `CUB_IGNORE_DEPRECATED_API` so that we can still test deprecated stuff.